bears/js/JSHintBear.py

```diff
@@ -16,19 +17,388 @@ class JSHintBear(LocalBear, Lint):
+                                                .jshintrc file is used as the
+                                                configuration file.
         '''
         self.arguments = '--verbose {filename}'
+        options = {
+            "bitwise": prohibit_bitwise,
+            "freeze": prohibit_prototype_overwrite,
+            "curly": force_braces,
+            "eqeqeq": prohibit_type_coercion,
+            "futurehostile": future_hostile,
+            "notypeof": prohibit_typeof,
+            "es3": es3,
+            "forin": force_filter_forin,
+            "funcscope": allow_funcscope,
+            "iterator": iterator,
+            "noarg": prohibit_arg,
+            "nocomma": prohibit_comma,
+            "nonbsp": prohibit_non_breaking_whitespace,
+            "nonew": prohibit_new,
+            "undef": prohibit_undefined,
+            "singleGroups": prohibit_groups,
+            "varstmt": prohibit_variable_statements,
+            "asi": allow_missing_semicol,
+            "debug": allow_debugger,
+            "boss": allow_assignmentcomparisions,
+            "evil": allow_eval,
+            "globalstrict": allow_global_strict,
+            "plusplus": allow_increment,
+            "proto": allow_proto,
+            "scripturl": allow_scripturls,
+            "supernew": allow_singleton,
+            "validthis": allow_this_stmt,
+            "withstmt": allow_with_stmt,
+            "moz": using_mozilla,
+            "noyield": allow_noyield,
+            "eqnull": allow_eqnull,
+            "lastsemic": allow_last_semicolon,
+            "loopfunc": allow_func_in_loop,
+            "expr": allow_expr_in_assignments,
+            "esnext": use_es6_syntax,
+            "elision": use_es3_array,
+            "mootools": environment_mootools,
+            "couch": environment_couch,
+            "jasmine": environment_jasmine,
+            "jquery": environment_jquery,
+            "node": environment_node,
+            "qunit": environment_qunit,
+            "rhino": environment_rhino,
+            "shelljs": environment_shelljs,
+            "prototypejs": environment_prototypejs,
+            "yui": environment_yui,
+            "mocha": environment_mocha,
+            "module": environment_module,
+            "wsh": environment_wsh,
+            "worker": environment_worker,
+            "nonstandard": environment_nonstandard,
+            "browser": environment_browser,
+            "browserify": environment_browserify,
+            "devel": environment_devel,
+            "dojo": environment_dojo,
+            "typed": environment_typed,
+            "phantom": environment_phantom,
+            "maxerr": maxerr,
+            "shadow": shadow,
+            "unused": prohibit_unused,
+            "latedef": allow_latedef,
+            "esversion": es_version
+        }
+        if maxstatements != 0:
+            options["maxstatements"] = maxstatements
```

Comment:
could you rather write a function like:

```python
def nozero_or_none(value):
    value = int(value)
    if value == 0:
        return None
    return value
```

and use that as the type annotation for all the params where you need this so we can get rid of this code?